.github/workflows/ci.yml

```diff
@@ -15,19 +15,21 @@ jobs:
         os: [ubuntu-latest, macos-latest, windows-latest]
     runs-on: ${{ matrix.os }}
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
         with:
           submodules: recursive
-      - uses: actions/setup-python@v2
+      - uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
+          cache: 'poetry'
       - name: Run image
         uses: abatilo/actions-poetry@v2.3.0
       - name: Install
-        run: poetry install
+        run: poetry install && echo "$(poetry env info --path)/bin" >> $GITHUB_PATH
```

Comment:
Is this necessary?

Reply:
I copied it from here: https://app.shortcut.com/dobles/stories/space/37/cool-space